README.md

```diff
@@ -3,6 +3,49 @@
 A [Singer](https://singer.io) Tap to extract currency exchange rate
 data from [fixer.io](http://fixer.io).
 
+## How to use it
+
+### Install and Run
+
+First, make sure Python 3 is installed on your system or follow these
+installation instructions for [Mac] or [Ubuntu].
+
+First, get an access key from [fixer.io](http://fixer.io). You need at least
+the basic plan to get exchange rates from a base currency (such as USD).
+
+Convert `config.sample.json` to `config.json`; fill out your parameters.
+
+It's recommended to use a virtualenv:
+
+```bash
+ python3 -m venv ~/.virtualenvs/tap-fixerio
+ source ~/.virtualenvs/tap-fixerio/bin/activate
+ pip install -U pip setuptools
+ pip install -e '.[dev]'
+```
+
+Set up the `target-csv` virtual environment according to the instructions [here](https://github.com/singer-io/target-csv/blob/master/README.md).
+
+These commands will install `tap-fixerio`  with pip, and then run it:
+
+```bash
+  pip install target-csv
+  tap-fixerio
```

Comment:
@timvisher  regarding -c config.json, I actually changed a line of the py file to make that file the default:
https://github.com/singer-io/tap-fixerio/pull/5/files#diff-1e7ded0f3322243ae4acd23372fd3a3aR85

Reply:
Ah that's interesting. Sorry I missed that.

I think I'm actually going to recommend that we remove that default. It's definitely off the beaten path in terms of behavior and I think it implies that there's a standard place to put the config file when there really isn't one yet.

FWIW, I would in the README put `~/singer.io/tap_fixerio_config.json` as the recommended place to put the config for now. I think I like the idea generally of having a standard place to put the configs but I'm not ready to make the call yet on where that should be. Opening an issue on the getting-started repo is probably a good way to start that conversation. :)

Reply:
(to your first part) - that's fair. I actually added it because this README didn't explain how to use -c so I thought "ah, this must be a mistake, I shouldn't need to pass in the config file."
But if the README says to, the default shall be removed.